src/bio/bio_internal.h

```diff
@@ -19,7 +19,7 @@
 
 #define BIO_DMA_PAGE_SHIFT	12	/* 4K */
 #define BIO_DMA_PAGE_SZ		(1UL << BIO_DMA_PAGE_SHIFT)
-#define BIO_XS_CNT_MAX		48	/* Max VOS xstreams per blobstore */
+#define BIO_XS_CNT_MAX		DSS_MAX_VOS_TGT_CNT	/* Max VOS xstreams per blobstore */
```

Comment:
DAOS server engine is on the top layer, we can't use engine stuff in BIO. This restriction could be defined in BIO or SMD.